src/Simpleue/Locker/MemcachedLocker.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Simpleue\Locker;
+
+class MemcachedLocker extends LockerAbstract
+{
+    /**
+     * @var \Memcached;
+     */
+    private $memcached;
+
+    public function __construct(\Memcached $memcached)
+    {
+        parent::__construct();
+
+        $this->memcached = $memcached;
+        $this->memcached->setOptions([
+            \Memcached::OPT_TCP_NODELAY => true,
+            \Memcached::OPT_NO_BLOCK => true,
+            \Memcached::OPT_CONNECT_TIMEOUT => 60
+        ]);
+    }
+
+    public function getLockerInfo()
+    {
+        return 'Memcached ( ' . json_encode($this->memcached->getServerList()) . ' )';
+    }
+
+    public function lock($job, $timeout = 30)
+    {
+        if (!$job) {
+            throw new \RuntimeException('Job for lock is invalid!');
+        }
+        $status = $this->memcached->add(
+            $this->getJobUniqId($job),
+            time() + $timeout + 1,
+            $timeout
+        );
+        if ($status) {
```

Comment:
"add" method from Memcached returns a boolean value, why not return it directly?

```
return $status
```
or 
```
return $this->memcached->add(
    $this->getJobUniqId($job),
    time() + $timeout + 1,
    $timeout
);
```